connman/src/wispr.c

```diff
@@ -896,6 +911,7 @@ static void proxy_callback(const char *proxy, void *user_data)
 					xml_wispr_parser_callback, wp_context);
 
 	wispr_portal_request_portal(wp_context);
+	wispr_portal_context_unref(wp_context);
```

Comment:
Wasn't there a commit earlier that added the "or proxy is null" part to that midfunction return?
If that was needed, it would suggest that this does get called with context!=NULL && proxy==NULL -> is it ok to skip context unref in such cases?

Reply:
Urgh, no idea. In case of error the wispr_portal_request_portal() does unref already and apparently it increases the reference when called. I've been trying to understand these but to no success...

Reply:
2f928223f927c4012393b75e38f0ce3ba9b47545 should deal with this.